### PR TITLE
Add Circle Linux profile to Anaconda(rhel-9)

### PR DESCRIPTION
**Pull request by bella485**
The PR adds CircleLinux to the rhel-9 branch of anaconda.

### CODE REVIEW

**Comment by bella485** [comment]
Please let me know if additional work or information is required for commit. Thanks!

File: data/product.d/circle.conf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+[Product]
+product_name = Circle Linux
+
+[Base Product]
+product_name = Red Hat Enterprise Linux
+
+[Anaconda]
+# List of enabled Anaconda DBus modules for RHEL.
+#  but without org.fedoraproject.Anaconda.Modules.Subscription
+kickstart_modules =
```
[the remark this answers is not in the document]
Comment (bella485) [reply data/product.d/circle.conf]: OK, i had new a bug https://bugzilla.redhat.com/show_bug.cgi?id=2095351  and forbidden_modules instead pushed.

**Comment by jkonecny12** [comment]
All the ACKs are now on place. This could be set to release.

**Pull request merged**